docs/en-US/commands/Get-JiraIssueWorklog.md

```diff
@@ -0,0 +1,105 @@
+---
+external help file: JiraPS-help.xml
+Module Name: JiraPS
+online version: https://atlassianps.org/docs/JiraPS/commands/**Get-JiraIssueWorklog**/
```

Comment:
remote the `*` around the name of the function

Reply:
Done, sorry about that!